docs/translations/translations.json

```
@@ -289,6 +289,7 @@
     "/customization/theme": "Theme",
     "/customization/theming": "Theming",
     "/customization/palette": "Palette",
+    "/customization/dark-mode": "Dark Mode",
```

The current heading convention

```suggestion
    "/customization/dark-mode": "Dark mode",
```